fill/actuators/actuator.h

```diff
@@ -0,0 +1,17 @@
+#ifndef ACTUATOR_H
+#define ACTUATOR_H
+
+class Actuator {
+public: 
+    /* Actuates the actuator. 
+    Returns true if activation is successful, and false otherwise. 
+    Sets isActuating to true if activation is successful.*/  
+    virtual bool Actuate() = 0;
```

Comment:
I feel like this should be set to false for readability

Reply:
I am not sure if that is allowed in my version of the C++ standard. (it wants it to just be =0). I don't know why this is the case, but I am getting the red squiggles in my IDE

Reply:
I don't think the 0 is the return value if that's what you're thinking. I think it is a special case that when a virtual function = 0, then it has to be implemented by subclasses. I see the = 0 convention a lot in C++

Reply:
Yep, just realized that